### DISCUSSION TITLE
Application size

### DISCUSSION THREAD

**Discussion by UmarFarooq-Ch** [tutao/tutanota#7418] [General]
I just noticed the application size got doubled, is tuta also going to make the app bigger as it is with the other clients, like gmail??

before the update, on version: 232.240621.0
![IMG_0384](https://github.com/user-attachments/assets/a37436f8-b388-4efe-be95-0cfbcfd133b7)

after the update, on version: 240.240813.0
![IMG_0385](https://github.com/user-attachments/assets/cba415e6-4f41-4bfb-9367-f49ca23a18dc)

**Comment by charlag** [comment]
Hey, we will check to be sure but it is probably native code that we added to support notifications.
We are still a relatively tiny app compared to most iOS apps
